roles/opensrp/templates/opensrp_local_checkout_path/configs/assets/config/opensrp.properties.j2

```diff
@@ -116,3 +116,14 @@ opensrp.cors.max.age={{ opensrp_cors_max_age }}
 
 #Team module settings
 use.opensrp.team.module={{ opensrp_use_opensrp_team_module }}
+
+
+# S3 configuration (should be populated for deployments using S3 multimedia storage)
+aws_access_key_id=
+aws_secret_access_key=
+aws_region=
+aws_bucket=
+aws_key_folder=
+
+# multimedia file manager (is file system by default)
+multimedia_file_manager=FileSystemMultimediaFileManager
```

Comment:
Add variables to each of these configs
 
for the file manager default the variable on the OpenSRP role

Reply:
Yea, this was just a quick fix to enable deployment.

Working on more comprehensive issue [here](https://github.com/OpenSRP/playbooks/issues/154).

Reply:
Done.